examples/resources/awscc_stepfunctions_activity/stepfunctions_activity.tf

```diff
@@ -0,0 +1,3 @@
+resource "awscc_stepfunctions_activity" "sfn_activity" {
+  name = "my-activity"
+}
```

Comment:
Did you want to add the tag?
tags = [{
    key   = "Modified By"
    value = "AWSCC"
  }]

Reply:
added 🚀